### PR TITLE
fix: move install mysql-client to ci image

### PR DESCRIPTION
**Pull request by tabVersion**
Signed-off-by: tabVersion <tabvision@bupt.icu>

I hereby agree to the terms of the [Singularity Data, Inc. Contributor License Agreement](https://gist.github.com/skyzh/0663682a70b0edde7ae991492f2314cb#file-s9y_cla).

## What's changed and what's your intention?

fix main

aca70301a066b29f133a0ca205bb85e89a237a1c

## Checklist

~~- [ ] I have written necessary rustdoc comments~~
~~- [ ] I have added necessary unit tests and integration tests~~
~~- [ ] I have added fuzzing tests or opened an issue to track them. (Optional, recommended for new SQL features).~~
~~- [ ] All checks passed in `./risedev check` (or alias, `./risedev c`)~~

## Refer to a related PR or issue link (optional)

the issue is introduced because of the upstream change

### CODE REVIEW

File: ci/docker-compose.yml [hunk 1]
```diff
@@ -30,28 +30,28 @@ services:
       retries: 5
 
   source-test-env:
-    image: public.ecr.aws/x5u3w5h6/rw-build-env:v20230119-1
+    image: public.ecr.aws/x5u3w5h6/rw-build-env:v20230126
     depends_on:
       - mysql
```
Comment (jon-chuang) [inline ci/docker-compose.yml:35]: Hmm, seems that we have dependency on another mysql docker image... was it necessary/is it still necessary?

Comment (jon-chuang) [reply ci/docker-compose.yml:35]: Oh I see, we are using the client to talk to this service? Makes sense.

**Review by jon-chuang — APPROVED** [review]
LGTM

File: ci/docker-compose.yml (hunk 1 above)
Comment (tabVersion) [reply ci/docker-compose.yml:35]: The problem is that we did not update apt source before installing `mysql-client`

**Pull request merged**